core/integration/container_test.go

```diff
@@ -932,6 +932,115 @@ func TestContainerEnvVariablesReplace(t *testing.T) {
 	require.Contains(t, res.Container.From.WithEnvVariable.WithExec.Stdout, "GOPATH=/gone\n")
 }
 
+func TestContainerWithLabel(t *testing.T) {
```

Comment:
nit: Feel free to use the Go SDK in these tests. The reason the other tests are not using the SDK it's just because they were written before the SDK existed. For new tests though, the SDK is preferred

Reply:
Done, great idea. As we found out about this bug: https://github.com/dagger/dagger/issues/4398